docs/the_black_code_style/current_style.md

```diff
@@ -260,16 +260,6 @@ If you are adopting _Black_ in a large project with pre-existing string conventi
 you can pass `--skip-string-normalization` on the command line. This is meant as an
 adoption helper, avoid using this for new projects.
 
-(labels/experimental-string)=
-
-As an experimental option (can be enabled by `--experimental-string-processing`),
```

Comment:
We could mention `--preview` explicitly in the style doc, not just in the future style. Maybe not here, perhaps at the very top?

Reply:
Yes, let's do that